services/app/src/components/chessBoard/Square.jsx

```diff
@@ -64,10 +64,10 @@ const Square = ({
             <img src={figures[item.figure].img} alt={item.figure} className={classNames(["figure-img", key > 0 && "figure-img-nth"])} />
           </div>
         ))}
-        {filteredFigures.length === 1 && <span className="figure-text">{`${items[0].score} SP`}</span>}
+        {filteredFigures.length === 1 && <span className="figure-text">{`${PIECES.find(piece => piece.name === items[0].figure).strength}`}</span>}
```

Comment:
Can you please give more details about how this solves the problem?

Also, do you need to use `.find` here? There's an object called `figures` in scope, which could be used to get figures without iterating through the array e.g.
```
  `${figures[items[0].figure}`
```